vector/build.gradle

```diff
@@ -132,6 +136,26 @@ android {
             buildConfigField "String", "FLAVOR_DESCRIPTION", "\"GooglePlay\""
         }
 
+        act {
+            applicationId "au.gov.act.hd.aether"
```

Comment:
You should just check with Jasen/Mark regarding the applicationId, I think the intention was that anything AETHER-specific was either separated out to deployment/build rather than be part of the GitHub code.

Reply:
Rafi the intention is that only pegacorn communicate code goes in github.  Aether code goes in DevOps (which can source from github too).  But I can this applicationId be "net.fhirfactory.pegacorn.communicate" ? Or is it describing AETHER specific functionality?

Reply:
I can add "net.fhirfactory.pegacorn.communicate" as application id (android) and bundle identifier (iOS). I think this id should also carry some information about the organization, usually the reverse domain name. we can change it anytime before we upload it to the store.

Reply:
Hi Rafi, we will need net.fhirfactory.pegacorn.communicate in the github source code and AETHER in the DevOps git source code.  The current thinking is that only the skinning/theming elements will be in the AETHER in the DevOps git source code, everything else will be in github.

Reply:
Hi Jasen, this id is basically application ID like https://play.google.com/store/apps/details?id=com.facebook.orca , this is the URL of facebook messenger and id/bundle identifier of this app is com.facebook.orca

Reply:
Hi Rafi, we will eventually end up with two application ids

1. Communicate which will have an application id like net.fhirfactory.pegacorn.communicate and will be specified in github

1. Lingo which can be au.gov.act.hd.lingo but this will be specified in the ACT Health git repository for Lingo